SPECS/ksh/ksh.spec

```
@@ -32,20 +25,20 @@ BuildRequires:  sudo
 Requires:       coreutils
 Requires:       diffutils
 
-Requires(post): coreutils
-Requires(post): grep
-Requires(post): systemd
-
+Requires(post): grep, coreutils, systemd
 Requires(postun): sed
 
+Provides: /bin/ksh
+Provides: /bin/rksh
+
 %description
 KSH-93 is the most recent version of the KornShell by David Korn of
 AT&T Bell Laboratories.
 KornShell is a shell programming language, which is upward compatible
 with "sh" (the Bourne Shell).
 
 %prep
-%autosetup -n %{name}-%{verBetaFull} -p1
+%autosetup -n %{name}-%{version}
```

Not needed anymore. The default is `%{name}-%{version}` anyway.
```suggestion
%autosetup
```